microservice_templates/microservice_python3_template/supervisor/_MICROSERVICE_PYTHON3_TEMPLATE_.conf

```diff
@@ -0,0 +1,3 @@
+[program:_MICROSERVICE_PYTHON3_TEMPLATE_]
+directory=/opt/_MICROSERVICE_PYTHON3_TEMPLATE_/src
+command=python3 _MICROSERVICE_PYTHON3_TEMPLATE_.py 80
```

Comment:
The port is already defined in _MICROSERVICE_PYTHON3_TEMPLATE_.py:
`run(host='0.0.0.0', port=80)`